python/lsst/ap/association/diaPipe.py

```diff
@@ -109,6 +109,31 @@ def __init__(self, *, config=None):
         if not config.doWriteAssociatedSources:
             self.outputs.remove("associatedDiaSources")
 
+    def adjustQuantum(self, datasetRefMap: pipeBase.InputQuantizedConnection):
+        """Override to make adjustments to `lsst.daf.butler.DatasetRef` objects
+        in the `lsst.daf.butler.core.Quantum` during the graph generation stage
+        of the activator.
+
+        This implementation checks to make sure that the filters in the dataset
+        are compatible with AP processing as set by the Apdb/DPDD schema.
```

Comment:
Because I'm using the name of the task in dax_apdb. Not sure it there is a standard for how to name stuff like so I will go with what the interface object is called.